package.json

```diff
@@ -23,6 +23,7 @@
     "test.local.sims.app": "wdio tests/configs/wdio.local.ios.sims.app.conf.ts",
     "test.local.sims.web": "wdio tests/configs/wdio.local.ios.sims.web.conf.ts",
     "test.local.desktop.multi": "wdio tests/configs/wdio.local.desktop.multi.conf.ts",
+    "test.local.desktop.storybook": "wdio tests/configs/wdio.local.desktop.storybook.conf.ts --storybook",
```

Comment:
This requires a storybook instance to be running somewhere right? Is this needed? Should we either remove this or create a minimal storybook example? Or rename the script to note that this is an example.

Reply:
You are correct, I can change the url to a sample the is open on the internet